### PR TITLE
Reverse has one

### PR DESCRIPTION
**Pull request by anditsung**
add relation to test HasOne field

### CODE REVIEW

File: database/migrations/2018_07_16_180920_create_addresses_table.php [hunk 1]
```diff
@@ -21,6 +21,7 @@ public function up()
             $table->string('state');
             $table->string('postal_code');
             $table->string('country');
+            $table->unsignedInteger('user_id');
```
Comment (crynobone) [inline database/migrations/2018_07_16_180920_create_addresses_table.php:24]: This going to break other tests which doesn't expect to set `user_id` when testing address.

File: app/Nova/Address.php [hunk 2]
```diff
@@ -45,6 +46,8 @@ public function fields(Request $request)
                 Text::make('Postal Code'),
                 Text::make('Country'),
             ]),
+
+            BelongsTo::make('user', 'user', User::class),
```
Comment (crynobone) [inline app/Nova/Address.php:50]: Should be optional. Otherwise might be best to create new resources and models to demonstrate the usage.

**Review by crynobone — CHANGES_REQUESTED** [review]

**Comment by anditsung** [comment]
got it. will create a new resource and model for this test

**Comment by anditsung** [comment]
hi, i already create new resource and model

**Pull request merged**